[2024-12-11T17:30:00.662+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-11T17:30:00.687+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_orchestrator.test_connection.dbt_test_connection scheduled__2024-12-11T17:15:00+00:00 [queued]>
[2024-12-11T17:30:00.694+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_orchestrator.test_connection.dbt_test_connection scheduled__2024-12-11T17:15:00+00:00 [queued]>
[2024-12-11T17:30:00.694+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-11T17:30:00.705+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): test_connection.dbt_test_connection> on 2024-12-11 17:15:00+00:00
[2024-12-11T17:30:00.710+0000] {standard_task_runner.py:63} INFO - Started process 695 to run task
[2024-12-11T17:30:00.713+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_orchestrator', 'test_connection.dbt_test_connection', 'scheduled__2024-12-11T17:15:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp3dhfp3fw']
[2024-12-11T17:30:00.714+0000] {standard_task_runner.py:91} INFO - Job 90: Subtask test_connection.dbt_test_connection
[2024-12-11T17:30:00.756+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_orchestrator.test_connection.dbt_test_connection scheduled__2024-12-11T17:15:00+00:00 [running]> on host 49f8df2c9427
[2024-12-11T17:30:00.834+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ma7moud' AIRFLOW_CTX_DAG_ID='dbt_orchestrator' AIRFLOW_CTX_TASK_ID='test_connection.dbt_test_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-12-11T17:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-11T17:15:00+00:00'
[2024-12-11T17:30:00.835+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-11T17:30:00.845+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-11T17:30:00.846+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt debug --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt/ecommerce']
[2024-12-11T17:30:00.857+0000] {subprocess.py:86} INFO - Output:
[2024-12-11T17:30:02.526+0000] {subprocess.py:93} INFO - [0m17:30:02  Running with dbt=1.8.0
[2024-12-11T17:30:02.536+0000] {subprocess.py:93} INFO - [0m17:30:02  dbt version: 1.8.0
[2024-12-11T17:30:02.537+0000] {subprocess.py:93} INFO - [0m17:30:02  python version: 3.9.19
[2024-12-11T17:30:02.537+0000] {subprocess.py:93} INFO - [0m17:30:02  python path: /home/***/.local/bin/python
[2024-12-11T17:30:02.538+0000] {subprocess.py:93} INFO - [0m17:30:02  os info: Linux-6.8.0-49-generic-x86_64-with-glibc2.36
[2024-12-11T17:30:02.598+0000] {subprocess.py:93} INFO - [0m17:30:02  Using profiles dir at /opt/***/dbt
[2024-12-11T17:30:02.599+0000] {subprocess.py:93} INFO - [0m17:30:02  Using profiles.yml file at /opt/***/dbt/profiles.yml
[2024-12-11T17:30:02.599+0000] {subprocess.py:93} INFO - [0m17:30:02  Using dbt_project.yml file at /opt/***/dbt/ecommerce/dbt_project.yml
[2024-12-11T17:30:02.600+0000] {subprocess.py:93} INFO - [0m17:30:02  adapter type: postgres
[2024-12-11T17:30:02.601+0000] {subprocess.py:93} INFO - [0m17:30:02  adapter version: 1.8.0
[2024-12-11T17:30:02.671+0000] {subprocess.py:93} INFO - [0m17:30:02  [[33mWARNING[0m]: Deprecated functionality
[2024-12-11T17:30:02.671+0000] {subprocess.py:93} INFO - The `tests` config has been renamed to `data_tests`. Please see
[2024-12-11T17:30:02.672+0000] {subprocess.py:93} INFO - https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax for more
[2024-12-11T17:30:02.672+0000] {subprocess.py:93} INFO - information.
[2024-12-11T17:30:02.719+0000] {subprocess.py:93} INFO - [0m17:30:02  Configuration:
[2024-12-11T17:30:02.720+0000] {subprocess.py:93} INFO - [0m17:30:02    profiles.yml file [[32mOK found and valid[0m]
[2024-12-11T17:30:02.720+0000] {subprocess.py:93} INFO - [0m17:30:02    dbt_project.yml file [[32mOK found and valid[0m]
[2024-12-11T17:30:02.721+0000] {subprocess.py:93} INFO - [0m17:30:02  Required dependencies:
[2024-12-11T17:30:02.733+0000] {subprocess.py:93} INFO - [0m17:30:02   - git [[32mOK found[0m]
[2024-12-11T17:30:02.733+0000] {subprocess.py:93} INFO - 
[2024-12-11T17:30:02.734+0000] {subprocess.py:93} INFO - [0m17:30:02  Connection:
[2024-12-11T17:30:02.735+0000] {subprocess.py:93} INFO - [0m17:30:02    host: 172.18.0.3
[2024-12-11T17:30:02.736+0000] {subprocess.py:93} INFO - [0m17:30:02    port: 5432
[2024-12-11T17:30:02.736+0000] {subprocess.py:93} INFO - [0m17:30:02    user: ***
[2024-12-11T17:30:02.737+0000] {subprocess.py:93} INFO - [0m17:30:02    database: ecommerce
[2024-12-11T17:30:02.738+0000] {subprocess.py:93} INFO - [0m17:30:02    schema: oltp
[2024-12-11T17:30:02.739+0000] {subprocess.py:93} INFO - [0m17:30:02    connect_timeout: 10
[2024-12-11T17:30:02.739+0000] {subprocess.py:93} INFO - [0m17:30:02    role: None
[2024-12-11T17:30:02.740+0000] {subprocess.py:93} INFO - [0m17:30:02    search_path: None
[2024-12-11T17:30:02.741+0000] {subprocess.py:93} INFO - [0m17:30:02    keepalives_idle: 0
[2024-12-11T17:30:02.741+0000] {subprocess.py:93} INFO - [0m17:30:02    sslmode: None
[2024-12-11T17:30:02.742+0000] {subprocess.py:93} INFO - [0m17:30:02    sslcert: None
[2024-12-11T17:30:02.742+0000] {subprocess.py:93} INFO - [0m17:30:02    sslkey: None
[2024-12-11T17:30:02.743+0000] {subprocess.py:93} INFO - [0m17:30:02    sslrootcert: None
[2024-12-11T17:30:02.744+0000] {subprocess.py:93} INFO - [0m17:30:02    application_name: dbt
[2024-12-11T17:30:02.744+0000] {subprocess.py:93} INFO - [0m17:30:02    retries: 1
[2024-12-11T17:30:02.745+0000] {subprocess.py:93} INFO - [0m17:30:02  Registered adapter: postgres=1.8.0
[2024-12-11T17:30:02.898+0000] {subprocess.py:93} INFO - [0m17:30:02    Connection test: [[32mOK connection ok[0m]
[2024-12-11T17:30:02.898+0000] {subprocess.py:93} INFO - 
[2024-12-11T17:30:02.898+0000] {subprocess.py:93} INFO - [0m17:30:02  [32mAll checks passed![0m
[2024-12-11T17:30:04.013+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-11T17:30:04.014+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-11T17:30:04.035+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_orchestrator, task_id=test_connection.dbt_test_connection, run_id=scheduled__2024-12-11T17:15:00+00:00, execution_date=20241211T171500, start_date=20241211T173000, end_date=20241211T173004
[2024-12-11T17:30:04.054+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-11T17:30:04.083+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-11T17:30:04.084+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
